pandaserver/dataservice/adder_atlas_plugin.py

```diff
@@ -237,324 +235,321 @@ def update_outputs(self):
+                            tmp_dest_list = [DataServiceUtils.getDestinationSE(
+                                file.destinationDBlockToken)] if DataServiceUtils.getDestinationSE(
+                                file.destinationDBlockToken) else \
+                                [destination_se_site_spec.setokens_output[scope_dst_se_site_spec_output].get(
+                                    file.destinationDBlockToken,
+                                    destination_se_site_spec.ddm_output[scope_dst_se_site_spec_output])]
+                        elif file.destinationDBlockToken in ["", None, "NULL"]:
+                            # use default endpoint
+                            tmp_dest_list = [src_site_spec.ddm_output[scope_src_site_spec_output]]
+                        elif (
+                            DataServiceUtils.getDestinationSE(file.destinationDBlockToken)
+                            and src_site_spec.ddm_output[scope_src_site_spec_output] == destination_se_site_spec.ddm_output[scope_dst_se_site_spec_output]
+                        ):
+                            tmp_dest_list = [DataServiceUtils.getDestinationSE(file.destinationDBlockToken)]
+                            # RSE is specified
+                        elif DataServiceUtils.getDistributedDestination(file.destinationDBlockToken):
+                            tmp_dest_list = [DataServiceUtils.getDistributedDestination(file.destinationDBlockToken)]
+                            dist_datasets.add(file_destination_dispatch_block)
+                            # RSE is specified for distributed datasets
+                        elif (
+                            src_site_spec.cloud != self.job.cloud
+                            and (not src_site_spec.ddm_output[scope_src_site_spec_output].endswith("PRODDISK"))
+                            and (self.job.prodSourceLabel not in ["user", "panda"])
+                        ):
+                            # T1 used as T2
+                            tmp_dest_list = [src_site_spec.ddm_output[scope_src_site_spec_output]]
+                        else:
+                            tmp_dest_list = []
+                            tmp_se_tokens = src_site_spec.setokens_output[scope_src_site_spec_output]
+                            for tmp_dest_token in file.destinationDBlockToken.split(","):
+                                if tmp_dest_token in tmp_se_tokens:
+                                    tmp_dest = tmp_se_tokens[tmp_dest_token]
+                                else:
+                                    tmp_dest = src_site_spec.ddm_output[scope_src_site_spec_output]
+                                if tmp_dest not in tmp_dest_list:
+                                    tmp_dest_list.append(tmp_dest)
+                        # add
+                        dataset_destination_map[file_destination_dispatch_block] = tmp_dest_list
+
+                # extra meta data
+                if self.ddm_backend == "rucio":
```

Comment:
AFAIU the ddm_backend was a transition variable from LFC/DQ2/Rucio, but right now there is only rucio (sometimes the backend is not set explicitly, but it's defaulted to Rucio). Do we need these if's?

Reply:
[root@aipanda098 panda]# grep "ddm backend " panda-adder.log | grep -v rucio
<nothing>

Reply:
So there is also potential to remove the stuff around line 70